custom parameter types support

koichirok opened the pull request:
Hi, 

I want to use cucumber's custom parameter type definition feature, so I implemented it in cytorus.

This PR contains 2 commits:

* 1st commit adds and exports `defineParameterType` function with the same interface as cucumber-js
   * I export it from `src/Registry.js`, but would it be better to export from `src/Globals.js` ?
* 2nd commit adjusts the loading order of the `.js` (or `.ts`) files to allows users to define their own parameter types anywhere under the `cypress/integration` directory.

Please have a look.

amitguptagwl commented:
@koichirok Yes, it'll be good to have common place to import library functions. So `Global.js` is recommended.

koichirok commented:
> @koichirok Yes, it'll be good to have common place to import library functions. So `Global.js` is recommended.

Done. changed `defineParameterType` to export from `Global.js` and moved its type definitions to `Global.d.ts`

> Can you please also merge the code to `pr-cucumber-type` branch? So I can check it offline and merge into master.

Merged, but could not push because it did not seem to be allowed.

```
$ git push upstream pr-cucumber-type
remote: Permission to NaturalIntelligence/cytorus.git denied to koichirok.
fatal: unable to access 'https://github.com/NaturalIntelligence/cytorus.git/': The requested URL returned error: 403
```

amitguptagwl commented:
Thanks. Let me verify the changes in local
